toolbox/util/gcsClient.go

```diff
@@ -61,15 +60,23 @@ func (gcs *GCSClient) GetFileFromGCSString(bucket, obj string) (string, error) {
 	}
 	defer func() {
 		if err = r.Close(); err != nil {
-			log.Printf("Failed to close gcs file reader, %v", err)
+			log.Printf("Failed to close gcs file reader, %v\n", err)
 		}
 	}()
-
 	buf := new(bytes.Buffer)
 	if _, err = buf.ReadFrom(r); err != nil {
-		log.Printf("Failed to read from gcs reader, %v", err)
+		log.Printf("Failed to read from gcs reader, %v\n", err)
 		return "", err
 	}
-
 	return buf.String(), nil
 }
+
+// WriteTextToFileOnGCS writes text to file on gcs
```

Comment:
rename to WriteTestToFile

Reply:
I believe you mean `WriteTextToFile`. and done.